backend/seeder.ts

```diff
@@ -4,7 +4,7 @@ import products from './data/products'
 import { UserModel } from './models/userModel'
 import Product from './models/productModel'
 import Order from './models/orderModel'
-import connectDB from './db/connectDb'
+import { connectDB } from './db/connectDb'
 
 dotenv.config()
```

Comment:
don't forget to remove `dotenv.config` from every file that's not the `config` file.

Reply:
good eyes hahahah removed!